frontend/components/input/SelectionField.tsx

```diff
@@ -0,0 +1,130 @@
+import { useState } from "react";
+import { Check, ChevronDown } from "react-feather";
+
+import { Text } from "@/components/Text";
+import { Icon, IconName } from "@/components/graphics/Icon";
+
+export type SelectionListItem = {
+	title: string;
+	description: string;
+	isSelected: boolean;
+	icon: IconName;
+};
+
+interface SelectionFieldProps {
+	/**
+	 * Title
+	 */
+	title: string;
+	/**
+	 * Selection list
+	 */
+	list: SelectionListItem[];
+	/**
+	 * Additional classes
+	 */
+	className?: string;
+}
+
+/**
+ * UI component for displaying a SelectionField
+ */
+export const SelectionField = ({
+	title,
+	list,
+	className = "",
+	...props
+}: SelectionFieldProps) => {
+	const [visible, setVisible] = useState(false);
+	const [selected, setSelected] = useState<SelectionListItem>(
+		getListSelected(list)
+	);
+	return (
+		<div>
+			<Text className="mb-1 text-sm font-semibold text-neutral-400">
+				{title}
+			</Text>
+			<div
+				className="flex	hover:cursor-pointer"
+				role="button"
+				onKeyDown={() => {}}
```

Comment:
I think this is a change resulting from a sonarcloud issue. Im not sure about the requirement here, but I think `combobox` would be the proper choice for the role. So the result should be something like this:
```suggestion
				role="combobox"
				onKeyUp={clickOnEnter}
				tabIndex={0}
```
Note that you can use the util function `clickOnEnter` to dispatch a click event on pressing enter.

PS: You can do something similar for the option with role `option`.